source/Calamari/Kubernetes/Conventions/GatherAndApplyRawYamlConvention.cs

```diff
@@ -135,19 +129,15 @@ private IEnumerable<Resource> ApplyBatchAndReturnResources(int index, string glo
                 var token = JToken.Parse(outputJson);
 
                 List<Resource> lastResources;
-                if (token.Type == JTokenType.Array)
-                {
-                    lastResources = token.ToObject<List<Resource>>();
-                }
-                else if (token["kind"]?.ToString() != "List" ||
+                if (token["kind"]?.ToString() != "List" ||
                     (lastResources = token["items"]?.ToObject<List<Resource>>()) == null)
                 {
                     lastResources = new List<Resource> { token.ToObject<Resource>() };
                 }
 
                 foreach (var resource in lastResources)
                 {
-                    log.Info($"'{resource.Kind}/{resource.Metadata.Name}' created.");
```

Comment:
Removing this punctuation means the output is the same as the old deployment scripts